docs_src/design/adr/system-management/0006-EdgeX-Metrics.md

```diff
@@ -0,0 +1,93 @@
+
+# EdgeX Metrics Collection  
+
+## Status 
+
+**proposed** - for Hanoi release
+
+## Requirements
+System Management services currently provide a limited set of “metrics” to requesting clients.  Namely, it provides requesting clients with service CPU and memory usage; both metrics about the resource utilization of the service itself versus metrics that are about what is happening inside of the service.  Arguably, the current system management metrics can be provided by the container engine and orchestration tools (example: by Docker engine) or by the underlying OS tooling.
+
+Going forward, users of EdgeX will want to have more insights – that is more metrics – on what is happening in the data collection, device management and monitoring, and event handling processes inside of Edge services.  The collection of this type of metric may require service level instrumentation relevant to capture and send data (what has been termed control plane data) about relevant EdgeX operations.  EdgeX does not currently offer any service instrumentation. 
+
+### Requested Metrics
+**General**
+- Latency statistics – from device sensor up north and back(and vice - versa).
+- Throughput of events
+- Indication of health – that events are being processed
+- Validation failures
+- Data/events not transiting the system; being blocked somewhere and queueing up
+- The volume of data each edge system is producing north bound.
```

Comment:
Would be helpful to elaborate with an example reference (or two) of what an `"edge system"` could be (perhaps as a parenthetical note).